ui/js/values.js

```diff
@@ -32,6 +32,8 @@ treeherder.value("thPlatformNameMap", {
     "android-4-4": "Android 4.4",
     "android-4-4-armv7-api11": "Android 4.4 API11+",
     "android-5-0-armv7-api11": "Android 5.0 API11+",
+    "b2gdroid-4-0-armv7-api11": "B2GDroid 4.0 API11+",
+    "android-4-0-armv7-api11-partner1": "Android 4.0 API11+ partner1",
```

Comment:
On Wed, Oct 14, 2015 at 12:09 PM, Jordan Lund <notifications@github.com>
wrote:

> In ui/js/values.js
> <https://github.com/mozilla/treeherder/pull/1063#discussion_r42038456>:
>
> > @@ -32,6 +32,8 @@ treeherder.value("thPlatformNameMap", {
> >      "android-4-4": "Android 4.4",
> >      "android-4-4-armv7-api11": "Android 4.4 API11+",
> >      "android-5-0-armv7-api11": "Android 5.0 API11+",
> > +    "b2gdroid-4-0-armv7-api11": "B2GDroid 4.0 API11+",
> > +    "android-4-0-armv7-api11-partner1": "Android 4.0 API11+ partner1",
>
> thanks for the review.
>
> these are an android variant. think of them like a partner repack. we plan
> to have many of these variants and this one is only one example. It's
> actually a proof of concept. so we have been using names like 'dummy apk',
> 'partner sample 1', etc.
>
> They are fennec builds that checkout a third party repo and use a special
> mozconfig. they are based on 4.0 and api-11+ right now.
>
>    - I think we could afford to be less specific (drop the '4.0 api11+')
>    in the buildername assuming there is no push back from
>       - we could also normalize partner1 to something like prtnr1
>
> both of these assume there is no pushback from mobile and @ncalexan
> <https://github.com/ncalexan>
>

This is fine, or I'm happy to shorten if we must.